bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "coffee-script",
   "main": [
-    "lib/coffee-script/coffee-script.js"
+    "lib/coffee-script/browser.js"
```

Comment:
This is fine. I’ve used Bower, it’s only for frontend (browser-based) JavaScript.